ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,399 @@
+specification of Zarr. The changes related to core specification follow a
+strict and thorough review process and should be adopted by everyone in the
+Zarr community.
+
+- Core protocol ZEP
+
+Describes a ZEP which involves changes in the core specification of Zarr. Core
+protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply to
+[Zarr Specifications] and its various implementations under the
+[zarr_implementations] GitHub repo. The core protocol ZEPs should be adopted by
+every implementation of Zarr and, in general, the overall Zarr community. Core
+ZEPs must go through a thorough review process, including involvement,
+discussion, and voting from the author of the proposed ZEP, Zarr Steering
+Council, author(s) of various Zarr implementations, and open-source
+projects/research groups using Zarr and the general Zarr community. The general
+advice is that everyone should be made aware of changes introduced by core
+ZEPs. Core protocol ZEPs require community consensus, and consumers are typically not
+free to ignore them.
+
+Note: Currently, Specification ZEPs only deal with the core storage
+specification of Zarr. The scope of SPEC ZEPs will be extended in the future.
+
+> Informational ZEP
+
+Describes a ZEP design issue or provides general guidelines or information to
+the Zarr community but does not propose a new feature. Informational ZEPs do
+not necessarily represent Zarr community consensus or recommendation, so users
+and implementers are free to ignore informational ZEPS or follow their advice.
+
+> Process ZEP
+
+Describes a new process around Zarr and its implementations or proposes a
+change to (or an event in) a process. They may propose an implementation, but
+not to Zarr’s codebase; they require community consensus; unlike informational
+ZEPs, they are more than recommendations, and users are typically not free to
+ignore them. Examples include procedures, guidelines, changes to the
+decision-making process, and changes to the tools or environment used in Zarr’s
+specification development. Any meta-ZEP is also considered a Process ZEP.
+
+## ZEP Workflow
+
+The ZEP process begins with a new idea for Zarr. It is highly recommended that
+a single ZEP contain a single key proposal or new idea. Small enhancements or
+patches often don’t need a ZEP and can be injected into the Zarr development
+workflow with a pull request to the Zarr ZEP repo. The more focused the ZEP,
+the more successful it tends to be. If in doubt, split your ZEP into several
+well-focused ones.
+
+Each ZEP must have a champion -- someone who writes the ZEP using the style and
+format described below, shepherds the discussions in the appropriate forums,
+and attempts to build community consensus around the idea. The ZEP champion
+(a.k.a. Author) should first attempt to ascertain whether the idea is suitable
+for a ZEP. Posting to [Gitter] or creating an issue on the [community] repository
```

Comment:
Let's clarify that ZEPs can have multiple co-authors.

Reply:
Added a separate line for this. ^
Done. 💯